<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixCalculator</class>
 <widget class="QWidget" name="MatrixCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="bnt_load_nodes">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Nodes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QFrame" name="frm_los">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="bnt_loadprofiles">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Loads an already fetched elevation profile</string>
        </property>
        <property name="text">
         <string>Load Profiles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_getprofiles">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Connects to Google Elevation API to get the profiles of all the links (saves the result)</string>
        </property>
        <property name="text">
         <string>Get Profiles (all)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_getprofile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Connects to Google Elevation API to get the profile of the selected link. (won't be saved)</string>
        </property>
        <property name="text">
         <string>Get Profile (single)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_calculatelos_all">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Checks all the LoS of all the links in the Link Matrix that passed the threshold.</string>
        </property>
        <property name="text">
         <string>Calculate LoS (all)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_images_all">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>If enabled the Calculate LoS(all) will also generate a PDF with the links profile</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 175, 74);</string>
        </property>
        <property name="text">
         <string>with images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_alllos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>If enable the output will have all the links, with and without LoS. </string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 175, 74);</string>
        </property>
        <property name="text">
         <string>All Links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_calculatelos_single">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Checks the LoS of a single selected link in the Link Matrix.</string>
        </property>
        <property name="text">
         <string>Calculate LoS (single)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_images_single">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>If enabled the Calculate LoS(single) check will generate a PDF file</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 175, 74);</string>
        </property>
        <property name="text">
         <string>with images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_optimize_heights">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Optimize Height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_generatekml">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Generates a KML with the links of the Link Matrix</string>
        </property>
        <property name="text">
         <string>Generate KML</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_save_excel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>saves the Link Matrix to a CSV file</string>
        </property>
        <property name="text">
         <string>Save LoS Excel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_fileid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 175, 74);</string>
        </property>
        <property name="text">
         <string>File Identifier Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_fileprefix">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The output file will have in its name the written identifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_apikey">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 175, 74);</string>
     </property>
     <property name="text">
      <string>Google API Key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="txt_load_pops">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QTreeWidget" name="tbl_linksmatrix">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lbl_threshold">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(50, 175, 74);</string>
     </property>
     <property name="text">
      <string>Threshold Distance</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="bnt_load_pops">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load POPS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="txt_api">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Google Elevation API Key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="bnt_calculate_links">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Calculates a link matrix between POPs and Nodes with a link distance threshold set by the spinbox</string>
     </property>
     <property name="text">
      <string>Calculate Links</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="4" colspan="2">
    <widget class="QListWidget" name="lst_log">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>134</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>134</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="txt_load_nodes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="spn_distance">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Distance threshold to consider it a link</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> km</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.040000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
